ZAPD/ZDisplayList.cpp

```diff
@@ -810,6 +810,13 @@ void ZDisplayList::Opcode_G_VTX(uint64_t data, int i, std::string prefix, char*
 		sprintf(line, "gsSPVertex(@r, %i, %i),", nn, _SHIFTR(hi, 17, 7));
 	}
 
+	// Hack: Don't extract vertices from a unknown segment.
+	if (!Globals::Instance->HasSegment(GETSEGNUM(data))) {
+		segptr_t segmented = data & 0xFFFFFFFF;
+		references.push_back(segmented);
+		parent->AddDeclaration(segmented, DeclarationAlignment::Align16, 16, "Vtx", StringHelper::Sprintf("0x%08X", segmented), "");
+		return;
+	}
```

Comment:
I dont follow whats happening here, if the data is from an unknown segment how come you need to do references.pushback and add a declaration?

only asking cause this same fix needs to be implemented in the gfxdis branch

Reply:
i implemented a different fix here: https://github.com/fig02/ZAPD/blob/gfxdis/ZAPD/ZDisplayList.cpp#L1453
If you find the time to try it out with object_fhg, let me know if it works